project/AppDependencies.scala

```diff
@@ -2,7 +2,7 @@ import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  val bootStrapPlayVersion = "6.2.0"
+  val bootStrapPlayVersion = "7.0.0"
```

Comment:
align version here as in the other project to 7.1

Reply:
done